jetson/percep/main.cpp

```diff
@@ -160,6 +161,7 @@ int main() {
     #endif
     
     #endif
+    cout << "bearing: " << obstacleMessage.bearing << " right bearing: " << obstacleMessage.rightBearing << endl;
```

Comment:
is there a reason to leave this in? looks like a debug print statement

Reply:
Deleted